### PR TITLE
individual page working not styled with comments

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,4 +31,8 @@ config/initializers/secret_token.rb
 bower.json
 
 # Ignore pow environment settings
+
 .powenv
```
Comment (f3r) [inline .gitignore]: Please remove, as you are not using https://github.com/basecamp/pow afaik

File: app/assets/javascripts/angular/controllers/series/show.js.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+app.controller('SeriesShowCtrl', ['$scope', '$http', '$location', '$routeParams', function($scope, $http, $location, $routeParams){
+
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/series/show.js.erb]: Please check: https://github.com/WDI-HK-7/findmetv/pull/7/files#diff-42e805002727ca0c39d869976171df60R3

either you add an extra slash everywhere or in none:

``` js
var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
```

vs

``` js
var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
```

I'd recommend for you to add it in your Mac Environment first:

``` sh
export URL="http://localhost:3000/"
```

and then, directly reference it everywhere:

``` js
$http.get(<%= ENV['URL'] %> + 'series/' + $routeParams.id).success(function(response){
```

File: app/assets/javascripts/angular/controllers/static_pages.js.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// Note: you have to duplicate $scope in 2 places to make it work
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/static_pages.js.erb:1]: see: https://docs.angularjs.org/guide/di

File: app/controllers/series_controller.rb [hunk 4]
```diff
@@ -2,13 +2,19 @@ class SeriesController < ApplicationController
...
     @series = Serie.all
   end
+
   def show
     @serie = Serie.find_by_id(params[:id])
-
     if @serie.nil?
       render :json => {
-        :message => { :message => "Cannot find dish" }
+        :message => { :message => "Cannot find serie -- yeah" }
```
Comment (f3r) [inline app/controllers/series_controller.rb]: Pls remove

File: .gitignore [hunk 5]
```diff
@@ -31,4 +32,6 @@ config/initializers/secret_token.rb
 bower.json
 
 # Ignore pow environment settings
```
Comment (f3r) [inline .gitignore]: Pls remove comment as well :cactus:

File: app/assets/javascripts/angular/controllers/static_pages.js.erb (hunk 3 above)
Comment (ghost) [inline app/assets/javascripts/angular/controllers/static_pages.js.erb:1]: I'm not showing static pages anywhere.

File: .gitignore (hunk 5 above)
Comment (ghost) [inline .gitignore]: Is this it?

File: app/assets/javascripts/angular/controllers/static_pages.js.erb (hunk 3 above)
Comment (f3r) [inline app/assets/javascripts/angular/controllers/static_pages.js.erb:1]: I just want you to understand WHY you have to duplicate $scope... it's called `dependency injection` ;)

**Comment by f3r** [comment]
be kind

https://www.atlassian.com/git/tutorials/rewriting-history/git-commit--amend

File: app/assets/javascripts/angular/controllers/series/index.js.erb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+app.controller('SeriesIndexCtrl', ['$scope', '$http', '$location', function($scope, $http, $location){
+
+  $http.get("<%= ENV['URL'] %>" + '/series').success(function(response){
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/series/index.js.erb]: if you have a `localhost:3000/` and you add `/series`... you are going to get a double slash: `localhost:3000//series' ...

**Pull request merged**